Added animation in texts

coderbysoul opened the pull request.

coderbysoul commented:
https://github.com/Codess-Cafe/Codess-Website/assets/96008596/59c53a65-a57b-463d-b5fd-37f282791b27

Issue #28

VidhiBhatt01 reviewed (changes requested):
@coderbysoul, this animation does not look good. Can we try something else?

coderbysoul commented:
Shall I change both the animations?

On Wed, 18 Oct, 2023, 11:03 am VIDHI BHATT, ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> @coderbysoul <https://github.com/coderbysoul>, this animation does not
> look good. Can we try something else?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Codess-Cafe/Codess-Website/pull/34#pullrequestreview-1684058029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW4PTFFTS4Z2GA4IOIMYJY3X75S37AVCNFSM6AAAAAA6EL4ZEOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOBUGA2TQMBSHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

coderbysoul commented:
Ok, I will work on it

coderbysoul commented:
I have modified the text animations for the Come be a part.. line and updated my pull request

https://github.com/Codess-Cafe/Codess-Website/assets/96008596/f96775e1-2d47-4f34-9316-1888f7dac883

VidhiBhatt01 commented on index.html:
```diff
@@ -213,8 +214,31 @@ <h1 class="big-heading">
                   Community of Empowered Women in Tech
                 </h1>
                 <h5 class="big-heading">
-                  Come be a part of a vibrant community of high achieving,
```
Women not woman

Keep it original only no need to add any animation

VidhiBhatt01 replied:
Just keep the first animation on this PR (for the line community of empowered...), no need to add any animation for second line. Keep it original

coderbysoul replied:
Removed the animations from the second line, can you please check

VidhiBhatt01 commented on app.js:
```diff
@@ -50,4 +50,5 @@ function clearActiveClasses() {
-}
\ No newline at end of file
+}
+
```
nit: extra line

coderbysoul replied:
I have written the code in app.js from line 55 to 76 for the animations in the second line, which I have deleted now, so this extra line(Line 54) is no longer there. Line 53 is the last line of code in app.js now

VidhiBhatt01 commented on index.html:
```diff
@@ -204,7 +204,8 @@
         </nav>
-
+        
+        
```
nit: extra line

coderbysoul replied:
Removed and a made another commit to this pr just now

VidhiBhatt01 commented:
Also add a video with only updated changes

coderbysoul commented:
> Also add a video with only updated changes


https://github.com/Codess-Cafe/Codess-Website/assets/96008596/99f8a377-c1f8-4a1d-8adf-9c08653e97fa

Resolved: Issue #34 
Previous: No Animation in the big heading (h1) of hero section
Changes: Added Animation to the big heading(h1) of hero section

VidhiBhatt01 reviewed (changes requested):
pull the latest changes and resolve merge conflicts

coderbysoul commented:
> pull the latest changes and resolve merge conflicts

I have resolved the conflicts in my latest commit
